libheif/api/libheif/heif.h

```diff
@@ -1966,6 +1980,13 @@ struct heif_decoded_mastering_display_colour_volume
   double min_display_mastering_luminance;
 };
 
+struct heif_ambient_viewing_environment
+{
+  uint32_t ambient_illumination;
+  uint16_t ambient_light_x;
+  uint16_t ambient_light_y;
+};
+
```

Comment:
Is this struct finalized in the standard? I did not find its definition. If it's not final yet, we should move it to `heif_experimental.h`.

Reply:
The property is in ISO/IEC 23008-12:2022 Section 6.5.36 "Ambient viewing Environment". That just points back to ISO/IEC 14496-12:2022, Section 12.1.9 "Ambient viewing environment" for the syntax and semantics.

ISO/IEC 14496-12:2022 has the syntax, but references ISO/IEC 23008-2 for the semantics. The semantics are in ISO/IEC 23008-2:2023 Section D.3.39.

Reply:
Thanks for guiding me through this jungle of documents.

Since the `ambient_viewing_environment` and `content_colour_volume` are defined in H.265 without any versioning information, I think we can dare to use it in the API without a version number. If it would be an HEIF property only, I would have proposed to use a version number in the struct.